packages/postgres-database/tests/test_utils_services_limitations.py

```diff
@@ -0,0 +1,239 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+# pylint: disable=too-many-arguments
+from dataclasses import fields
+from typing import Awaitable, Callable
+
+import pytest
+from aiopg.sa.connection import SAConnection
+from aiopg.sa.result import RowProxy
+from faker import Faker
+from simcore_postgres_database.models.groups import user_to_groups
+from simcore_postgres_database.utils_services_limitations import (
+    ServiceLimitationsCreate,
+    ServiceLimitationsOperationNotAllowed,
+    ServiceLimitationsOperationNotFound,
+    ServicesLimitationsRepo,
+)
+
+
+@pytest.fixture
+def random_service_limitations(
+    faker: Faker,
+) -> Callable[[int, int | None], ServiceLimitationsCreate]:
+    def _creator(gid: int, cluster_id: int | None) -> ServiceLimitationsCreate:
+        return ServiceLimitationsCreate(
+            gid=gid,
+            cluster_id=cluster_id,
+            ram=faker.pyint(),
+            cpu=faker.pydecimal(),
```

Comment:
MINOR: check as well default value and unlimited

Reply:
hmm, well the database does not care, it will not test anything special. I think this might come at a later point.

Reply:
ok